packages/core/src/dialects/abstract/query-interface.types.ts

```diff
@@ -6,16 +6,28 @@ export interface QueryInterfaceOptions {
   queryGenerator: AbstractQueryGenerator;
 }
 
-/** Options accepted by {@link AbstractQueryInterfaceTypeScript#createSchema} */
+/** Options accepted by {@link AbstractQueryInterface#createSchema} */
```

Comment:
Any specific reason you updated this? It is defined in the AbstractQueryInterfaceTypeScript

Reply:
Yes, otherwise the typedoc fails as the `AbstractQueryInterfaceTypeScript` is not exported.
I exported the query-interface.types.ts and query-generator.types.ts files from the abstract directory so that they can be used as the types were previously in the query-interface.d.s and query-generator.d.ts files.

If you'd prefer to leave the types in the query-interface.d.s and query-generator.d.ts files, I'll revert these changes.

Reply:
Ah yes, I forgot about that. Then this looks good to me. Thanks for the explanation!